kivymd/uix/snackbar.py

```diff
@@ -44,34 +44,34 @@ def build(self):
 .. image:: https://github.com/HeaTTheatR/KivyMD-data/raw/master/gallery/kivymddoc/snackbar-simple.gif
     :align: center
 
-Usage with padding
+Usage with snackbar_x, snackbar_y
 ------------------
 
 .. code-block:: python
 
-    Snackbar(text="This is a snackbar!", padding="20dp").open()
+    Snackbar(text="This is a snackbar!", snackbar_x="10dp", snackbar_y="10dp", size_hint_x=(Window.width - (dp(10) * 2)) / Window.width).open()
```

Comment:
Use PEP.

Reply:
Wouldn't be easier to just say x,y instead of snackbar_x, snackbar_y?

Reply:
Based on how displaying the Snackbar works for transitions from outside of the screen to appearing on the screen it has to be done this way. Those are the locations on the screen we want the Snackbar to appear. the snackbar.x and snackbar.y get set before animation into the screen to set where they are around the screen.